diff --git a/node_modules/react-native-reanimated/android/build.gradle b/node_modules/react-native-reanimated/android/build.gradle
index d82d9c7..54a7b31 100644
--- a/node_modules/react-native-reanimated/android/build.gradle
+++ b/node_modules/react-native-reanimated/android/build.gradle
@@ -245,7 +245,7 @@ def JS_RUNTIME = {
 
     // Check if Hermes is enabled in app setup
     def appProject = rootProject.allprojects.find { it.plugins.hasPlugin('com.android.application') }
-    if ((REACT_NATIVE_MINOR_VERSION >= 71 && appProject?.hermesEnabled?.toBoolean()) || appProject?.ext?.react?.enableHermes?.toBoolean()) {
+    if ((REACT_NATIVE_MINOR_VERSION >= 71 && appProject?.findProperty('hermesEnabled')?.toBoolean()) || appProject?.ext?.has('react') ? appProject?.ext?.react?.enableHermes?.toBoolean() : false) {
         return "hermes"
     }
 
} 